tests/perf.c

```diff
@@ -539,7 +663,7 @@ test_edit_batch_create(struct test_state *state, struct timespec *ts_start, stru
         }
     }
 
-    if ((r = sr_apply_changes(state->sess, state->count * 100))) {
+    if ((r = sr_apply_changes(state->sess, 0))) {
```

Comment:
If there is no special reason, `state->count * 100` or any other value depending on the input size should be used in all `sr_apply_changes()` calls.

Reply:
done